tasks.js

```diff
@@ -1,30 +1,115 @@
-module.exports = [{
-  name: 'covid-rdt-followup',
-  icon: 'icon-healthcare',
-  title: 'task.covid_followup.title',
-  appliesTo: 'contacts',
-  appliesToType: ['person'],
-  appliesIf: function (c) {
-    this.mostRecentRdt = Utils.getMostRecentReport(c.reports, 'covid_rdt');
-    return this.mostRecentRdt && Utils.getField(this.mostRecentRdt, 'test_result') === 'positive';
+
+module.exports = [
+  // Covid Rdt Followup
+  {
+    name: 'covid-rdt-followup',
+    icon: 'icon-healthcare',
+    title: 'task.covid_followup.title',
+    appliesTo: 'contacts',
+    appliesToType: ['person'],
+    appliesIf: function (c) {
+
+      this.mostRecentRdt = Utils.getMostRecentReport(c.reports, 'covid_rdt');
+      return this.mostRecentRdt && Utils.getField(this.mostRecentRdt, 'test_result') === 'positive';
+    },
+    resolvedIf: function (c, r, event) {
+      const startTime = Utils.addDate(event.dueDate(c, r), -event.start);
+      const endTime = Utils.addDate(event.dueDate(c, r), event.end + 1);
+
+      const reportsAfterRdt = c.reports.filter(report => report.reported_date >= this.mostRecentRdt.reported_date);
+      return Utils.isFormSubmittedInWindow(reportsAfterRdt, 'covid_rdt_followup', startTime, endTime);
+    },
+    events: [{
+      start: 1,
+      end: 3,
+      dueDate: function() {
+        return Utils.addDate(new Date(this.mostRecentRdt.reported_date), 1);
+      },
+    }],
+    actions: [{
+      type: 'contacts',
+      form: 'covid_rdt_followup',
+      label: 'task.covid_followup.title',
+    }],
   },
-  resolvedIf: function (c, r, event) {
-    const startTime = Utils.addDate(event.dueDate(c, r), -event.start);
-    const endTime = Utils.addDate(event.dueDate(c, r), event.end + 1);
-    
-    const reportsAfterRdt = c.reports.filter(report => report.reported_date >= this.mostRecentRdt.reported_date);
-    return Utils.isFormSubmittedInWindow(reportsAfterRdt, 'covid_rdt_followup', startTime, endTime);
+
+  // Cha verification
+  {
+    name: 'cha-signal-verification',
+    icon: 'icon-healthcare',
+    title: 'Cha verification',
+    appliesTo: 'contacts',
+    appliesToType: undefined,
+    appliesIf: function (c) {
+      const isCha = user.parent && user.parent.type === 'health_center';
+      this.mostRecent8 = Utils.getMostRecentReport(c.reports, '8');
```

Comment:
Was it intentional to leave out "CHA signal reporting" report here?

Reply:
i am not sure to understand, can you clarify ?

Reply:
Signals can be reported by either a CHW or a CHA. The CHA signal reporting form is used by the CHA to report signal. So if we are retaining the CHA signal reporting workflow, a verification task should be generated for the same.

Reply:
good catch ! but i'll merge anyway and create another ticket for it (https://github.com/medic/config-covid/issues/44)  as i'm not sure yet that's something we would like to include in the MVP ( the CHA reporting signals ), will clarify with jane and michael.